clients/drcachesim/drcachesim.dox.in

```diff
@@ -158,7 +158,7 @@ Some of the more important markers are:
 
 - #TRACE_MARKER_TYPE_TIMESTAMP - The marker value provides a timestamp for this point of the trace (in units of microseconds since Jan 1, 1601 UTC). This value can be used to synchronize records from different threads. It is used in the sequential analysis of a multi-threaded trace.
 
-- #TRACE_MARKER_TYPE_CPU_ID - The marker value contains the CPU identifier on which the subsequent records were collected. It is useful to help track thread migrations occurring during execution.
+- #TRACE_MARKER_TYPE_CPU_ID - The marker value contains the CPU identifier on which the subsequent records were collected. It is useful to help track thread migrations occurring during execution. This marker is output in the header of each trace buffer. Note that if the thread migrates to a different CPU due to preemption by kernel before a buffer is full, we do not output a new TRACE_MARKER_TYPE_CPU_ID marker to capture that event. However, we expect that to be rare.
```

Comment:
This sounds like the thread started execution on the specified cpu for sure, but might have migrated before the end: when it's really the opposite.  Reword to make that clearer?

Reply:
Done. PTAL.

Because the marker is in the header (which comes before the trace entries), and because we use the phrase "subsequent records", it's a little more confusing that the marker is accurate for the tail entries. I clarified though. Lmk if this works.

Reply:
Forgot to reply here: yes it looks good.